### PR TITLE
Initializing subprocess according to ldoktor

### PR DESCRIPTION
**Pull request by rdkdd**
Signed-off-by: Radek Duda <rduda@redhat.com>

### CODE REVIEW

File: spice/tests/vm_listen.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def run(vt_test, test_params, env):
         test_params['start_vm'] = "yes"
         if test_params['spice_port_closed'] == "yes":
             cmd = "nc -l %s" % test_params['spice_port']
-            nc_process = process.SubProcess(cmd)
+            nc_process = process.get_sub_process_klass(cmd)(cmd)
```
Comment (Andrei-Stepanov) [inline spice/tests/vm_listen.py]: We will re-write this.
calling function on returning object. (one)(one)

Let it stay open, until we have time to review.

**Comment by rdkdd** [comment]
Please take a lok @Andrei-Stepanov

**Comment by Andrei-Stepanov** [comment]
Fine. but change please: 

```
Signed-off-by: root root@spiceqe.brq.redhat.com
```

**Comment by rdkdd** [comment]
Done. @Andrei-Stepanov  can you merge now?

**Pull request merged**